backend/requirements.txt

```diff
@@ -1,3 +1,7 @@
 Flask
 flask-restful
 uwsgi
+flask-sqlalchemy
+flask-migrate
+psycopg2-binary
+Flask-Testing
```

Comment:
you can keep A-Z order, it's better with no order. Try to specify which version of the module you use! Right now you will get the newest module, which can provide more issues rather than making your life easier.